README.md

```diff
@@ -43,7 +43,7 @@ To run it in background (detached mode), just add `-d` option
 $ docker-compose up --build -d
 ```
 
-Create a `main` container and enter bash:
+To run `main` container in bash mode, just use below command:
```

Comment:
The description is still vague. you'd better add a summary to indicate there are two ways to start a container and use symbol to distinguish the two part. for example.
> You can start container by:
> - Start all service defined in docker-compose.yml using docker-compose up
> you code example here
> - run `main` container in bash mod
> you code example here

Reply:
OK, I updated readme.